mixer/cmd/mixs/cmd/server.go

```diff
@@ -72,6 +72,14 @@ func serverCmd(info map[string]template.Info, adapters []adapter.InfoFn, printf,
 	if err := serverCmd.PersistentFlags().MarkHidden("configIdentityAttributeDomain"); err != nil {
 		fatalf("unable to hide: %v", err)
 	}
+	serverCmd.PersistentFlags().StringVar(&sa.LivenessProbeOptions.Path, "livenessProbePath", "",
+		"Path to the file for the liveness probe.")
+	serverCmd.PersistentFlags().DurationVar(&sa.LivenessProbeOptions.UpdateInterval, "livenessProbeInterval", 0,
+		"Interval of updating file for the liveness probe.")
+	serverCmd.PersistentFlags().StringVar(&sa.ReadinessProbeOptions.Path, "readinessProbePath", "",
+		"Path to the file for the readiness probe.")
+	serverCmd.PersistentFlags().DurationVar(&sa.ReadinessProbeOptions.UpdateInterval, "readinessProbeInterval", 0,
+		"Interval of updating file for the readiness probe.")
 
 	// TODO: Remove all this stuff by the 0.5 release (don't forget all associated YAML templates and any other uses of these options in the code
```

Comment:
(OT) is this comment correct ?

Reply:
I'm not aware of this, but I'd say that's unrelated to this PR. Let's file an issue.

Reply:
filed: https://github.com/istio/istio/issues/2943